packages/@coorpacademy-player-store/src/actions/api/progressions.js

```diff
@@ -287,10 +288,12 @@ DispatchedAction => {
       PROGRESSION_FETCH_BESTOF_FAILURE
     ],
     task: () => Progressions.findBestOf(engine.ref, type, ref, progressionId),
-    bailout: pipe(
-      getBestScore,
-      score => score !== undefined && score >= 0
-    ),
+    bailout: force
```

Comment:
`bailout` is used to prevent requests if the data is already there, but in case of `bestScore`, we may need to refresh it to be sure to have the higher best score value and prevent displaying a `+xx stars` every time the  user finishes the same course without reloading the app to clean the state.